examples/httpd/convox.yml

```diff
@@ -1,4 +1,18 @@
+resources:
+  postgresdb:
+    type: postgres
+  rediscache:
+    type: redis
 services:
   web:
     image: httpd
     port: 80
+  resource-tester:
+    image: nightfury1204/resource-tester:0.0.1
+    resources:
+      - postgresdb:DB_URL
+      - rediscache:REDIS_URL
```

Comment:
Attach the resources to the `web` service, don't need to create an extra service for this.

It's better to use the build option instead of a pre-built image for tests.

```
image: nightfury1204/resource-tester:0.0.1
```

Reply:
I did went for  for the build option, but there is a issue: https://github.com/convox/issues-private/issues/594

I went for multi service so that it will be tested for multi service setup also

Reply:
Okay, great job catching this up 👍 

About the image, I don't think using your docker hub repository is a good option, maybe @heronrs has access to the convox repo on docker hub?